app/src/main/java/com/kickstarter/services/transformers/GraphQLTransformers.kt

```diff
@@ -200,6 +198,30 @@ fun rewardTransformer(
         .build()
 }
 
+fun simpleShippingRuleTransformer(simpleShippingRules: FullProject.SimpleShippingRulesExpanded): ShippingRule {
+    val id = decodeRelayId(simpleShippingRules.locationId()) ?: -1
+    val country = simpleShippingRules.country() ?: ""
+    val displayName = simpleShippingRules.locationName()
+
+    val location = Location.builder()
+        .id(id)
+        .country(country)
+        .displayableName(displayName)
+        .name(displayName)
+        .build()
+    val cost = simpleShippingRules.cost()?.toDoubleOrNull() ?: 0.0
```

Comment:
`toDoubleOrNull` if the string is not a double will return null, and default value will apply 0.0
`toDoubleOrNull` precision: rounds up from the 15th digit
Take a look at the test `test simpleShippingRuleTransformer provides appropriate shippingRule` where all those use cases are tested